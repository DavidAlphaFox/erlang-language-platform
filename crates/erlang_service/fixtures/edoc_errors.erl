%% @doc This module contains a backtick,
%%      which causes `EDoc to crash and burn
-module(edoc_errors).

%% @doc This is function 1
one() ->
    1.

%% @docc This is function two, containing a warning
two() ->
    2.
