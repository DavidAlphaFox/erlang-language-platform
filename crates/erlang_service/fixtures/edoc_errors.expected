MODULE_DOC


FUNCTION_DOCS
{}

EDOC_DIAGNOSTICS
[
    DocDiagnostic {
        severity: "error",
        line: 2,
        message: "`-quote ended unexpectedly at line 2",
    },
]


